### PR TITLE
docs(snap): remove replicated JWT instructions

### PR DESCRIPTION
**Pull request by MonicaisHer**
Signed-off-by: Mengyi <mengyi.wang@canonical.com>

If your build fails due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-ui-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

**Comment by MonicaisHer** [comment]
Could you please review it? Thanks! @farshidtz

File: snap/README.md [hunk 1]
```diff
@@ -89,35 +89,10 @@ $ sudo snap set edgexfoundry kuiper=on
 
 ### JWT token
 
-API Gateway is needed to enter UI secure mode. Before the API Gateway can be used, a user and group must be created, and a JWT access token generated. This can be accomplished via the `secrets-config` and  the `snap set` commands:
+API Gateway is needed to enter UI secure mode. Please generate the JWT token by following this 
+[instruction](https://github.com/edgexfoundry/edgex-go/blob/main/snap/README.md#jwt-tokens).
```
Comment (farshidtz) [inline snap/README.md:93]: Please replace with:
> A JWT Token is required to access the UI securely through the API Gateway. To generate a JWT token, please refer to [these instructions](https://github.com/edgexfoundry/edgex-go/blob/main/snap/README.md#jwt-tokens) .

Comment (MonicaisHer) [reply snap/README.md]: Updated. Thanks!

**Review by farshidtz — APPROVED** [review]
Thanks. It looks good now.

**Review by badboy-huaqiao — APPROVED** [review]
LGTM

**Pull request merged**